docs/advanced/document-store/index.md

```diff
@@ -201,3 +201,25 @@ da[0, 'text'] = 'hello'
 Obviously, a DocumentArray with on-disk storage is slower than in-memory DocumentArray. However, if you choose to use on-disk storage, then often your concern of persistence overwhelms the concern of efficiency.
  
 Slowness can affect all functions of DocumentArray. On the bright side, they may not be that severe as you would expect. Modern database are highly optimized. Moreover, some database provides faster method for resolving certain queries, e.g. nearest-neighbour queries. We are actively and continuously improving DocArray to better leverage those features. 
+
+
+## Benchmarking
+The script `scripts/benchmarking.py` benchmarks DocArray's supported Document Stores in 6 different operations:
+* Create (indexing Documents)
+* Read
+* Update
+* Delete
+* Find Document by vector (Nearest Neighbor Search)
+* Find Document by condition (apply filter)
+
+The results were conducted on a 4.5 Ghz Quad-Core i7700k processor with Python 3.8.5 and using the official docker 
+images of the storage backends. The docker images were allocated 40 GB of RAM.
+The indexing and query performance on 128-dim embeddings is the following (unit is second):
+
+| Backend  | Create    | Read    | Update  | Delete  | Find by vector | Find by condition |
+|----------|-----------|---------|---------|---------|----------------|-------------------|
+| Memory   | 0.05921   | 0.00007 | 0.00003 | 0.00003 | 0.67842        | 0.57762           |
```

Comment:
this number is not human readable, solution

- declare the metric at column header, e.g. Read (us/ms)
- use scientific repr, 1e-7